### PR TITLE
hard-coding drbd/notify-split-brain.sh script path and no place here

### PR DESCRIPTION
**Pull request by cl-lab-k**

### CODE REVIEW

File: files/private-chef-cookbooks/chef-ha-drbd/attributes/default.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 default['private_chef']['drbd']['shared_secret'] = "promisespromises"
 default['private_chef']['drbd']['device'] = "/dev/drbd0"
 default['private_chef']['drbd']['disk'] = "/dev/opscode/drbd"
+default['private_chef']['drbd']['notify_split_brain_sh'] = '/usr/lib/drbd/notify-split-brain.sh'
```
Comment (irvingpop) [inline files/private-chef-cookbooks/chef-ha-drbd/attributes/default.rb]: So this appears to change the default location for the script - I'm not sure that's what we want for all users, is it?

**Comment by sdelano** [comment]
This looks like something that we should definitely fix. We've triaged it into the `accepted-minor` milestone, and will perform some additional QA to verify that it works as expected for all users. There may be a solution that is more general for locating the install location of the drbd scripts, and we should explore those options.

**Comment by stevendanna** [comment]
@chef/lob @irvingpop I've checked RHEL5, RHEL6, and a couple versions of Ubuntu and this script lives in the same location on all of those platforms.  The only users I could see this breaking are those who have made manual changes in /opt/opscode, which I don't think is viable for us to support long term.

I originally thought that we should potentially vendor this script; however, we already depend on drbd as a stable, external dependency.  This script is mentioned in their documentation and I don't have too much of a problem of relying on it.

That is a long way to say :+1:.  I'll merge this today unless someone convinces me otherwise.

**Comment by irvingpop** [comment]
:+1: Thanks again @stevendanna for setting me straight :)

**Comment by stevendanna** [comment]
This was merged here: https://github.com/chef/opscode-omnibus/commit/16d04c128e16a7c86a7d022650ff2b903b02610c

**Pull request closed without merging**